### PR TITLE
Mark doc areas that should be fixed for AS3 with "FIXME AS3"

### PR DESCRIPTION
**Pull request by glasser**

### CODE REVIEW

File: docs/source/api/apollo-server.md [hunk 1]
```diff
@@ -279,6 +279,7 @@ An object containing configuration options for connecting Apollo Server to [Apol
 An array of [plugins](../integrations/plugins) to install in your server instance. Each array element can be either a valid plugin object or a zero-argument function that _returns_ a valid plugin object.
 
 In certain cases, Apollo Server installs some of its built-in plugins automatically (for example, when you provide an Apollo Studio API key with the `APOLLO_KEY` environment variable). For details, see the API references for these plugins: [usage reporting](./plugin/usage-reporting/), [schema reporting](./plugin/schema-reporting/), and [inline trace](./plugin/inline-trace/).
+FIXME AS3: More things are installable plugins now (cache control, landing pages) which should be documented on their own pages and mentioned here.
```
Comment (StephenBarlow) [inline docs/source/api/apollo-server.md:282]: I would like to reorganize plugin content and will include this in that work

File: docs/source/data/subscriptions.mdx [hunk 2]
```diff
@@ -5,6 +5,8 @@ description: Persistent GraphQL read operations
 
 import {ExpansionPanel} from 'gatsby-theme-apollo-docs';
 
+FIXME AS3: Whole file needs to be rewritten, as we don't have built in subscription support any more, and probably shouldn't recommend using `subscriptions-transport-ws` at this time.
```
Comment (StephenBarlow) [inline docs/source/data/subscriptions.mdx:8]: I will gladly help overhaul, but I definitely don't know the "right" way to get this functionality working in 3

File: docs/source/deployment/heroku.md [hunk 3]
```diff
@@ -4,6 +4,8 @@ sidebar_title: Heroku
 description: Deploying your GraphQL server to Heroku
 ---
 
+FIXME AS3: Walk through and test this whole file.
```
Comment (StephenBarlow) [inline docs/source/deployment/heroku.md:7]: This one has an open communityPR that I'm going to address shortly, and I can definitely tackle this one

File: docs/source/getting-started.mdx [hunk 4]
```diff
@@ -185,6 +185,7 @@ We're up and running!
 We can now execute GraphQL queries on our server. To execute our first query,
 we can use **Apollo Sandbox**.
 
+FIXME AS3: Should we instead tell folks to go to the printed localhost URL and click through the landing page to Sandbox?
```
Comment (StephenBarlow) [inline docs/source/getting-started.mdx:188]: Yep, can tackle that

File: docs/source/migration-engine-plugins.md [hunk 5]
```diff
@@ -2,6 +2,8 @@
 title: Migrating from the "engine" option
 ---
 
+FIXME AS3: We can probably remove this but also have the first step in the migration guide say "if you're using `engine:`, then before migrating to 2.0, stop setting `engine` based on this page in the v2 migration guide"
```
Comment (StephenBarlow) [inline docs/source/migration-engine-plugins.md:5]: SGTM

File: docs/source/testing/graphql-playground.mdx [hunk 6]
```diff
@@ -30,7 +30,7 @@ Sandbox automatically attempts to connect to a GraphQL server running at `http:/
 
 [GraphQL Playground](https://github.com/graphql/graphql-playground) is a graphical, interactive, in-browser GraphQL IDE, created by [Prisma](https://www.prisma.io/) and based on [GraphiQL](https://github.com/graphql/graphiql).
 
-(Note: This whole page needs to be updated for AS3 now that Playground is not the default.)
+(FIXME AS3: This whole page needs to be updated for AS3 now that Playground is not the default.)
```
Comment (StephenBarlow) [inline docs/source/testing/graphql-playground.mdx:33]: Yes this is on my list as well

File: docs/source/migration.md [hunk 7]
```diff
@@ -3,6 +3,7 @@ title: Migrating to v3.0
 description: How to migrate to Apollo Server 3.0
 ---
 
+FIXME AS3: Lots more is needed here.
```
Comment (StephenBarlow) [inline docs/source/migration.md:6]: I think this is the pinnacle of what I _shouldn't_ take the first stab at

File: docs/source/data/subscriptions.mdx (hunk 2 above)
Comment (glasser) [reply docs/source/data/subscriptions.mdx:8]: @trevor-scheer might also be interested in this.

File: docs/source/api/apollo-server.md (hunk 1 above)
Comment (glasser) [reply docs/source/api/apollo-server.md:282]: https://github.com/apollographql/apollo-server/pull/5348

File: docs/source/data/subscriptions.mdx (hunk 2 above)
Comment (glasser) [reply docs/source/data/subscriptions.mdx:8]: (At some point we should PR to `graphql-ws` updating their README.)

Comment (trevor-scheer) [reply docs/source/data/subscriptions.mdx:8]: Note to self: there's a github backed codesandbox to update here

File: docs/source/data/file-uploads.md [hunk 8]
```diff
@@ -5,6 +5,8 @@ description: Enabling file uploads in Apollo Server
 
 > Note: As of Apollo Server 3.0, in order to allow the `graphql-upload` package to evolve and live on its own, uploads are no longer enabled by default and are not an integrated part of Apollo Server.
 
+FIXME AS3: Need to verify that the code below still works. Should note that federation does not support uploads.
```
Comment (glasser) [inline docs/source/data/file-uploads.md:8]: I'll do this.

Comment (glasser) [reply docs/source/data/file-uploads.md:8]: Resolved in #5355

**Comment by glasser** [comment]
@StephenBarlow @trevor-scheer I've added all these `FIXME AS3`s as cards in the Docs column of https://github.com/apollographql/apollo-server/projects/6. (Which can be converted to issues as needed, eg to assign them.) Given that, I'm going to close this PR rather than continue to track the FIXMEs here; the PR can still be useful for context for those cards.

**Pull request closed without merging**